blockdevice/probe/probe.go

```diff
@@ -149,7 +149,7 @@ func probe(devpath string) (devpaths []string) {
 	// Start by opening the block device.
 	// If a partition table was not found, it is still possible that a
 	// file system exists without a partition table.
-	bd, err := blockdevice.Open(devpath)
+	bd, err := blockdevice.Open(devpath, blockdevice.WithMode(blockdevice.ReadonlyMode))
```

Comment:
I hope we don't write to probed devices later? (I don't know if there's any good chance to track through it?)

Reply:
I've checked the code, it doesn't look like we do. I'm going to run integration tests in Talos now to see if anything got broken.